### PR TITLE
Jetlife .md

### PR DESCRIPTION
**Pull request by williamstj2442**
i was here

### CODE REVIEW

**Comment by VikdemoneJr** [comment]
i was here

**Comment by williamstj2442** [comment]
i was here

**Comment by williamstj2442** [comment]
i was here

File: _posts/0000-01-02-VikdemoneJr/.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (williamstj2442) [inline _posts/0000-01-02-VikdemoneJr/.md]: im back

**Review by williamstj2442 — COMMENTED** [review]
changed

**Review by williamstj2442 — COMMENTED** [review]
changed

**Review by williamstj2442 — COMMENTED** [review]
changed

**Pull request merged**